### ISSUE TITLE
Option to disable card hover images

### ISSUE THREAD

**Issue by anandaguneratne** [kellyelton/OCTGN#1377]
Could an option be implemented (perhaps as a property of a card) to disable the card image that appears when a player hovers over the card? We are trying to use cards to make a modular board, and the images of the board tiles are distracting and unnecessary.

Thanks!

**Comment by kellyelton** [comment]
Honestly having locked cards be game board pieces wasn't the plan and is not t he purpose of them, so I'm not quite sure what you're asking makes sense.

**Comment by anandaguneratne** [comment]
Regardless of whether or not it was the intended purpose, it is a major benefit of having the option to lock cards, and will greatly enhance the Mage Wars module. If what I am asking does not make sense, I'll try to explain again:

Basically, what I want is a card property (for instance, card.hoverImage some other such name; it could also be accessed via some function card.hoverImage()) as a boolean. If card.hoverImage == false, then when the player hovers the mouse over the card, no image will appear on the screen as it usually does. Developers would be able to change this card property as desired.

I don't know how OCTGN handles showing the images when hovering over cards, so I don't know how feasible this is, but intuitively it seems like a simple switch. Is there a major obstacle to implementing this?

**Comment by kellyelton** [comment]
I understand what you want, I'm just saying that I don't know if I'm going to add those features for that purpose, and instead I think a specific way to just put images on the table makes more sense.

**Comment by anandaguneratne** [comment]
Okay; in that case, is there a way to create a modular map from separate tile images, or is this a feature that could be implemented? Or perhaps a way to stamp the image of a card into the table? That would actually be preferable to using cards as tiles, since then other cards can't get 'lost' under the board.

**Comment by kellyelton** [comment]
Well there are lots of reasons using cards for this purpose is hacky and not really a great fit. Such as the deck editor, all the extra meta data involved with cards etc.

We don't have a way to put images on the board yet, but it's something we could implement and it wouldn't take that much work, considering they don't have to be player moved, and don't have to have a controller and all that extra junk that goes with in play elements.

**Comment by brine** [comment]
yeah I think in the end we should just expand the game board functionality to allow them to be modular, that way OCTGN can expand into board games.

**Comment by sIKE23** [comment]
A counter argument is that these boards take up a lot of space. 1/3 of my nugget package is taken up by boards. Making them cards lets me move them over into the image pack. I can see my game easily out growing the under 10MB package size with all of the modular tiles that will be coming in future releases. 

Yes the idea was to support games that have different card sizes. However, games with Modular board piece like mine or Zombicide it are a natural fit for them to be represented as cards since you can now place markers on them with out having to hack the Markers to be cards which is also icy. 

The only the two side effects (so far)have been pointed out on this thread, the first is the hoover image with a mouse over and the second is the Deckbuilder. Late last year the ability to filter off sets that you don't want to see was added to the Deckbuilder which solves that problem.

If you push the modular board pieces back into the Game Def how will the game devs be able to provide players with board layout options? With them being cards, this can be all done with python quite deftly. 

I specifically asked for the card.anchor feature with Modular Board Cards in mind. 

However you wish to expand (modular) board functionality please keep us in the loop as we are actively interested in how the works and will be implementing it.....

**Comment by Soul1355** [comment]
yeah, I'm kinda with sike on this one, what he had in mind for the anchored cards makes a lot of sense to me, but I'm not really clear on what their purpose is as they are implemented. Any chance you could explain a use case they were designed for to help clear it up? I'm all for taking another approach for board pieces if there's a good reason to keep locked cards as is (building them from the ground up would also allow for more features), but the concept seems better suited for the likes of a game board to me.

**Comment by sIKE23** [comment]
The game I support is a board game, and recently introduced modular board tiles. Since OTCGN does not have the great support for multiple board types much less modular board tiles, we decided to make the modular board tiles in OCTGN from cards. Here are a couple of examples.

![image 10](https://cloud.githubusercontent.com/assets/5695175/9148273/355a02f4-3d3c-11e5-999e-73ba57a66660.jpg)
![image 9](https://cloud.githubusercontent.com/assets/5695175/9148274/3bb47418-3d3c-11e5-8058-a341ce6eeb00.jpg)
![image 5](https://cloud.githubusercontent.com/assets/5695175/9148275/3ed19810-3d3c-11e5-96b5-224538b306ba.jpg)
The second example highlights the issue we are trying to resolve, since the "board tiles" are now cards we now get the on hover zoom on the "board tiles" which are quite large ,compared to the rest of the "cards" in the game, and results in zoomed image covering large sections of the board and can be quite annoying as you move your mouse "across the board" as you can imagine. 

Optimally a optional argument would be added to the set data for the card much like "size". 
"hover =  off" at the simplest would be off the top of my head. The engine would then know not to pop up the zoom image of that particular card guid.

**Comment by Soul1355** [comment]
I'd be inclined to roll it into the locking really, but again, I'm not clear if there's another use for locking I'm just not seeing where you'd still want the hover image.

**Comment by sIKE23** [comment]
I am not sure what the locking is.....I want the hoover image on "game​
cards" but not on board tile cards.

**Comment by sIKE23** [comment]
By locking did you mean anchoring? If so that is an interesting thought. Then your question make absolute sense, is there ever a need for a hoover/zoom image on a card that is anchored? For my game no. I would disable hoover/zoom on board specific (and other "non-game") cards.

**Comment by Soul1355** [comment]
oh right, sorry, I'm bad at the lingo. :P anchoring is exactly what I meant; I just see the little icon and that's where my brain goes.
